Example/Pods/Local Podspecs/TSSwiftMarkdownParser.podspec.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "TSSwiftMarkdownParser",
```

Comment:
Why are you still calling it `TS` `SwiftMarkdownParser`? You COULD call it `RRMarkdownParser`? :)